extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -504,6 +498,6 @@ class MembershipsButtonEdit extends Component {
 }
 
 export default compose( [
-	withSelect( select => ( { postId: select( 'core/editor' ).getCurrentPostId() } ) ),
+	withSelect( selector => ( { postId: selector( 'core/editor' ).getCurrentPostId() } ) ),
```

Comment:
Needed to import `select` from `@wordpress/data` to pass text attribute through to child inner blocks when the plan was selected. This meant the linter didn't like naming the arg here `select` as well, given it was already defined in the outer scope.

Open to suggestions on a better approach than this.

Reply:
is it possible to get innerButtons in the withSelect instead? ie.

```javascript

withSelect( ( select, { clientId } )  => ( { 
	postId: select( 'core/editor' ).getCurrentPostId(),
	innerButtons:  select( 'core/editor' ).getBlocksByClientId( clientId )	
} ) )

```